src/js/pages/Intro/Intro.jsx

```diff
@@ -89,7 +87,15 @@ export default class Intro extends Component {
           <span className="medium">
             This is our best guess - feel free to change.
           </span>
-          <AddressBox {...this.props} saveUrl="/ballot" />
+          <AddressBox
+            externalUniqueId={this.props.externalUniqueId}
```

Comment:
Since the Intro Component is a page, it doesn't seem like any of the variables being passed in as `this.props` would be coming from the `App` routing page.